js-intro.js

```diff
@@ -19,43 +36,74 @@ var temp3 = 212
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
-
-
+const mult5 = (arrMult5) => {
+    let answerMult5 = []
+    for (let i=0; i<arrMult5.length; i++){
+    answerMult5.push(arrMult5[i] * 5)
+  }
+  return answerMult5
+}
+console.log(mult5(myNumbers1))
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
-
+const mapMult5 = (array) => {
+  let answerMapMult5 = array.map(value => value * 5)
+  return answerMapMult5
+    }
+console.log(mapMult5(myNumbers2))
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
-
+const noVowel = (noVowelArray) => {
+    let eachLetter = noVowelArray.split("")
+    let bangAeiou = eachLetter.filter(value => value !== "a" && value !== "e" && value !== "i" && value !== "o" && value !== "u")
```

Comment:
You can add a `.join("")` at the end of this line, and a `return` instead of creating a variable to save some lines of code :) This solution still totally works, just a way to add efficiency